rskj-core/src/test/java/co/rsk/peg/FederationTestUtils.java

```diff
@@ -19,15 +19,27 @@
 package co.rsk.peg;
 
 import co.rsk.bitcoinj.core.BtcECKey;
+import co.rsk.bitcoinj.core.NetworkParameters;
 import org.ethereum.crypto.ECKey;
 
 import java.math.BigInteger;
+import java.time.ZonedDateTime;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class FederationTestUtils {
+
+    public static Federation getFederation(Integer... federationMemberPks) {
+        return new Federation(
+                FederationTestUtils.getFederationMembersFromPks(federationMemberPks),
```

Comment:
Maybe we can ignore the `FederationTestUtils` ref since we're already in that class (not 100% sure of how it behaves when in static methods)

Reply:
Yes! my bad!

Reply:
Done!